examples/wallets/examples/keystores.rs

```diff
@@ -0,0 +1,43 @@
+use ethers::prelude::*;
+use ethers::core::rand::thread_rng;
+use ethers::signers::{LocalWallet};
+use dotenv::dotenv;
+
+use eyre::Result;
+
+
+#[tokio::main]
+async fn main() -> Result<()>{
+
+    dotenv().ok();
+
+    let api_key = std::env::var("API_KEY").expect("expected environmental variable");
+    let encryption_path = std::env::var("ENCYPTION_PATH").expect("expected environmental variable");
+    let decryption_path = std::env::var("DECRYPTION_PATH").expect("expected environmental variable");
+    let password = std::env::var("PASSWORD").expect("expected environmental variable");
+
+    let ws = Ws::connect(api_key)
+        .await
+        .expect("no connection");
+
+    println!("Is Connected: {}", ws.ready());
+
+    //generates a brand new keystore with key
+    //name your keystore file, set the path, set the password
+    //*NOTE* -- pls don't store your passwords in a plain text file, this is just an example 
+    let key_store = LocalWallet::new_keystore(encryption_path, &mut thread_rng(), &password, Some("my_encrypted_keys"))
+        .expect("key store fail");
+    
+    let (your_signing_key, _) = key_store; 
```

Comment:
Then you can remove this line

Reply:
Thank you! I merged your suggestions ❤️

Reply:
You might need to fix linting

Reply:
Linting worked locally, I think the bot doesn't like dotenv crate. So, I removed it and put placeholder values in